storage.js

```diff
@@ -0,0 +1,27 @@
+// Get references to the form inputs
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const messageInput = document.getElementById('message');
```

Comment:
- [ ] Hi as part of the requirements, you're supposed to preserve data in the form and redisplay it after the page reloads but currently, it's not working. I realized you are trying to get the element with id `name` in the form but in your HTML file you don't have any element of id `name` Please kindly fix it
<img width="1300" alt="image" src="https://user-images.githubusercontent.com/37901469/224303673-b43da8e4-feb8-4983-817f-48d7091c2166.png">